dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -202,6 +212,14 @@ open class Jpql : JpqlDsl {
         return Paths.path(this.toEntity(), property)
     }
 
+    /**
+     * Creates a path expression with the entity and property.
+     */
+    @SinceJdsl("3.1.0")
+    fun <T : Any, V> Entityable<T>.path(getter: KFunction1<T, @Exact V>): Path<V & Any> {
```

Comment:
when i first put `in` keyword. IDE show me the following warning.
It might that `KFunction1` has `in` variance unlike `KProperty1`.

```
Projection is redundant: the corresponding type parameter of KFunction1 has the same variance
```

Reply:
Aha, I see. Thank you for the confirmation.